<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <widget class="TitleWidget" name="title_widget" native="true"/>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="login_page">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>16</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>24</pointsize>
                </font>
               </property>
               <property name="text">
                <string>用户登录</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_6">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QWidget" name="widget_3" native="true">
               <layout class="QGridLayout" name="gridLayout">
                <item row="2" column="0">
                 <widget class="QLabel" name="label_3">
                  <property name="styleSheet">
                   <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
                  </property>
                  <property name="text">
                   <string>密码：</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="login_name"/>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_2">
                  <property name="styleSheet">
                   <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
                  </property>
                  <property name="text">
                   <string>用户名：</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLineEdit" name="login_pwd"/>
                </item>
                <item row="1" column="0">
                 <spacer name="verticalSpacer_5">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Fixed</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_4" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QCheckBox" name="checkBox_isRem">
                  <property name="styleSheet">
                   <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
                  </property>
                  <property name="text">
                   <string>记住密码</string>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>25</width>
                    <height>25</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>153</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QToolButton" name="toolButton_reg_2">
                  <property name="styleSheet">
                   <string notr="true">font: 12pt &quot;微软雅黑&quot;;
text-decoration: underline;
color: rgb(0, 0, 255);</string>
                  </property>
                  <property name="text">
                   <string>没有账号？立即注册</string>
                  </property>
                  <property name="autoRaise">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_5" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>74</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QToolButton" name="toolButton_login">
                  <property name="minimumSize">
                   <size>
                    <width>250</width>
                    <height>60</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">font: 18pt &quot;微软雅黑&quot;;
background-color: rgb(255, 255, 255);
color:rgb(0, 0, 0)</string>
                  </property>
                  <property name="text">
                   <string>登  录</string>
                  </property>
                  <property name="autoRaise">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>74</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="2">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>16</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="reg_page">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QWidget" name="widget_6" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>83</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QWidget" name="widget_8" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QWidget" name="widget_title" native="true">
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>24</pointsize>
                </font>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="layoutDirection">
                   <enum>Qt::LeftToRight</enum>
                  </property>
                  <property name="text">
                   <string>用户注册</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_7" native="true">
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <layout class="QGridLayout" name="gridLayout_3">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_name">
                  <property name="text">
                   <string>用户名：</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <spacer name="horizontalSpacer_6">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="0" column="2">
                 <widget class="QLineEdit" name="reg_name"/>
                </item>
                <item row="1" column="0" colspan="2">
                 <widget class="QLabel" name="label_nickname">
                  <property name="text">
                   <string>昵称：</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2">
                 <widget class="QLineEdit" name="reg_nickname"/>
                </item>
                <item row="2" column="0" colspan="2">
                 <widget class="QLabel" name="label_pwd">
                  <property name="text">
                   <string>密码：</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="2">
                 <widget class="QLineEdit" name="reg_pwd"/>
                </item>
                <item row="3" column="0" colspan="2">
                 <widget class="QLabel" name="label_pwd_again">
                  <property name="text">
                   <string>确认密码：</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="2">
                 <widget class="QLineEdit" name="reg_pwd_again"/>
                </item>
                <item row="4" column="0" colspan="2">
                 <widget class="QLabel" name="label_phone">
                  <property name="text">
                   <string>手机：</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="2">
                 <widget class="QLineEdit" name="reg_phone"/>
                </item>
                <item row="5" column="0" colspan="2">
                 <widget class="QLabel" name="label_email">
                  <property name="text">
                   <string>邮箱：</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="2">
                 <widget class="QLineEdit" name="reg_email"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_9" native="true">
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>16</pointsize>
                </font>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QToolButton" name="toolButton_reg">
                  <property name="text">
                   <string>注册</string>
                  </property>
                  <property name="autoRaise">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_7">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QToolButton" name="toolButton_cancel">
                  <property name="text">
                   <string>取消</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_9">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>82</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="set_page">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <spacer name="horizontalSpacer_12">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>105</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="widget_10" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QWidget" name="widget_13" native="true">
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="0">
              <widget class="QLabel" name="label_5">
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>24</pointsize>
                </font>
               </property>
               <property name="text">
                <string>服务器设置</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_10">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>43</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QWidget" name="widget_11" native="true">
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <widget class="QLabel" name="label_serv_addr">
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>服务器地址：</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <spacer name="horizontalSpacer_11">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>13</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="2">
              <widget class="QLineEdit" name="lineEdit_serv_addr"/>
             </item>
             <item row="1" column="0">
              <spacer name="verticalSpacer_9">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>18</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="2" column="0" colspan="2">
              <widget class="QLabel" name="label_serv_port">
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>服务器端口：</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QLineEdit" name="lineEdit_serv_port"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_11">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>43</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QWidget" name="widget_12" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QToolButton" name="toolButton_set_yes">
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>16</pointsize>
                </font>
               </property>
               <property name="text">
                <string>确定</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_10">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>120</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton_cancel_2">
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>16</pointsize>
                </font>
               </property>
               <property name="text">
                <string>取消</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_13">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>104</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitleWidget</class>
   <extends>QWidget</extends>
   <header location="global">mywidget/titlewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>login_name</tabstop>
  <tabstop>login_pwd</tabstop>
  <tabstop>checkBox_isRem</tabstop>
  <tabstop>toolButton_reg_2</tabstop>
  <tabstop>toolButton_login</tabstop>
  <tabstop>reg_name</tabstop>
  <tabstop>reg_nickname</tabstop>
  <tabstop>reg_pwd</tabstop>
  <tabstop>reg_pwd_again</tabstop>
  <tabstop>reg_phone</tabstop>
  <tabstop>reg_email</tabstop>
  <tabstop>toolButton_reg</tabstop>
  <tabstop>toolButton_cancel</tabstop>
  <tabstop>lineEdit_serv_addr</tabstop>
  <tabstop>lineEdit_serv_port</tabstop>
  <tabstop>toolButton_set_yes</tabstop>
  <tabstop>toolButton_cancel_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
